tidb-binlog/pump_client/selector.go

```diff
@@ -30,6 +30,9 @@ const (
 
 	// Score means choose pump by it's score.
 	Score = "score"
+
+	// Unique means will only use one pump.
+	Unique = "unique"
```

Comment:
use which pump？

Reply:
ok, I will use `local` to replace `unique`